hwdb.d/70-touchpad.hwdb

```diff
@@ -56,6 +56,13 @@ touchpad:usb:v05acp0265:*
 touchpad:usb:v044ep1221:*
  ID_INPUT_TOUCHPAD_INTEGRATION=external
 
+###########################################################
+# Lenovo
+###########################################################
+# Thinkpad L14 Gen1 (AMD)
+touchpad:usb:v06cbp0081:*
```

Comment:
No, this entry is not necessary. As I commented in the previous PR, please drop the change to this file.

Reply:
-> #26987.